webapp/components/interop.js

```diff
@@ -1036,11 +1025,12 @@ class Interop2022FeatureChart extends PolymerElement {
   static get observers() {
     return [
       'updateChart(feature, stable)',
+      'updateYearInfo(year)'
```

Comment:
Same here. We don't need an observer method since year is being passed in. It won't change unless the page is reloaded